symfony/3-nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl

```diff
@@ -0,0 +1,27 @@
+server {
+    listen 80;
+
+    root /app/web;
+
+    server_name localhost;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
+    }
+
+    location ~ ^/(app|app_dev|config)\.php(/|$) {
```

Comment:
obv changing the location to an env var would work, but might there be another way that's not too bad